.circleci/config.yml

```diff
@@ -74,6 +76,19 @@ jobs:
             git remote add ro https://github.com/fairwindsops/charts
             git fetch ro master
             ct lint --config scripts/ct.yaml --chart-yaml-schema scripts/schema.yaml --debug
+
+  insights:
+    docker:
+      - image: quay.io/reactiveops/ci-images:v11-stretch
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 18.09.3
```

Comment:
Curious why you're pinning the version here. That's not something I've seen in any of our other stuff.

Reply:
We probably don't need it in this case. At one point I had done that in order to use a newer version of Docker (Circle doesn't default to the latest)

Reply:
Good catch - copypasta

Reply:
Apparently you have to pin: https://circleci.com/docs/2.0/building-docker-images/#docker-version

Reply:
I don't see there where it says you have to pin. I see where it says you "can" pin. 

In the orb and every other CI file I have it is just `- setup_remote_docker`

Reply:
Ahhh I misunderstood, I tried to just remove the patch version. Done.